css/main.css

```diff
@@ -0,0 +1,614 @@
+  height: 33px;
+  padding-bottom: 4px;
+  font-weight: 600;
+}
+.header-btn {
+  display: inline-flex;
+  float: right;
+  margin-right: 1rem;
+}
+.header-links a {
+  margin-left: 1rem;
+}
+.header-links label {
+  margin-left: 1rem;
+  margin-top: 0.25rem;
+  font-weight: 600;
+}
+.buggedicon {
+  margin-top: 1px;
+}
+.header svg {
+  width: 12.2rem;
+  position: absolute;
+  padding-top: 0.6rem;
+  top: 0px;
+  left: 33.5rem;
+}
+
+.side-bar {
+  float: right;
+  width: 14em;
+  position: absolute;
+  bottom: -1500px;
+  right: 0;
+  padding: 2rem;
+  margin-right: 70px;
+}
+
+.article {
+  margin-bottom: 0.5rem;
+}
+
+.article hr {
+  border-width: 0.7px;
+  color: lightgrey;
+}
+
+.article h2 {
+  font-size: 13px;
+}
+.article-link {
+  display: flex;
+  align-items: center;
+  color: grey;
+  text-decoration: none;
+}
+
+.article-link:hover {
+  text-decoration: underline;
+}
+.article-link div {
+  width: 10em;
+}
+
+.article-link div img {
+  width: 50%;
+}
+.article-text {
+  font-size: 14px;
+  width: 10em;
+  margin-left: -65px;
+  font-weight: 500;
+}
+
+.article-button {
+  background-color: rgb(98, 136, 165);
+  color: white;
+  position: relative;
+  right: 12em;
+  top: 2em;
+  display: block;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-size: 0.8125rem;
+  font-weight: 600;
+  letter-spacing: 0.05em;
+  line-height: 1.5rem;
+  margin-left: auto;
+  margin-right: auto;
+  text-transform: uppercase;
+  width: 22%;
+  border-width: 1px;
+  border-style: solid;
+  border-color: rgb(50, 104, 145);
+  border-image: initial;
+  border-radius: 3px;
+  padding: 5px 0px;
+  transition: background-color 0.6s ease 0s;
+}
+
+.article-button-links {
+  position: relative !important;
+
+  top: -1em;
+}
+
+.body-paragraph {
+  text-align: center;
+}
+
+.body-paragraph p {
+  font-size: 1.2rem;
+  margin-bottom: 2rem;
+  text-align: left;
+}
+.body-paragraph-2 {
+  text-align: center;
+  margin-bottom: -15rem;
+}
+.body-paragraph-2 h2 {
+  font-size: 17px;
+  margin-left: -25rem;
+  font-weight: 900;
+}
+.body-paragraph-2 :nth-child(2) {
+  text-align: left;
+  font-size: 13px;
+
+  font-weight: 100;
+  color: grey;
+}
+.body-image {
+  width: 41rem;
+}
+.body-image-text {
+  position: relative;
+  top: -29em;
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+.image-text {
+}
+.image-text p {
+  width: 13rem;
+  font-size: 12px;
+  padding-right: 0.3rem;
+  text-align: left;
+}
+.image-text p span {
+  font-size: 12px;
+  font-weight: 700;
+}
+
+.text-normal {
+  font-weight: 400 !important;
+}
+
+.body-image-text-2 {
+  position: relative;
+  top: -19em;
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.documentbottom {
+  width: 80%;
+  margin: -1.6rem auto;
+}
+
+/* footer */
+
+.footer {
+  border-top: 2px solid rgb(221, 221, 221);
+  border-bottom: 2px solid rgb(221, 221, 221);
+  margin: 65rem auto 0 auto;
+
```

Comment:
Remove whitespace. Always your code well indented and free of unneeded whitespace.